Libplanet.Tools/Mpt.cs

```diff
@@ -0,0 +1,75 @@
+#nullable enable
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Cocona;
+using Libplanet.RocksDBStore;
+using Libplanet.Store.Trie;
+
+namespace Libplanet.Tools
+{
+    public class Mpt
+    {
+        [Command(Description = "Compare two ")]
+        public void Diff(
+            [Argument(Name = "KEY-VALUE-STORE-PATH")]
+            string keyValueStorePath,
+            [Argument(Name = "STATE-ROOT-HASH")]
+            string stateRootHashHex,
+            [Argument(Name = "OTHER-STATE-ROOT-HASH")]
+            string otherStateRootHashHex,
+            [Option]
+            string keyValueStoreType,
```

Comment:
This option name is too long.  It should have a short name (e.g., `-t`) as well.

Also how about renaming it to `kvStoreType`?